src/metldata/builtin_transformations/count_content_values/data_transform.py

```diff
@@ -85,14 +88,17 @@ def transform_class(
 
 def transform_resource(
     *,
-    referenced_resources: dict[ResourceId, Resource],
-    target_resource: Resource,
+    referenced_resources: dict[str, Any],
+    target_resource: dict,
     relation_name: str,
     instruction: CountContentValueInstruction,
 ):
     """Apply the count content value transformation to each resource of a class."""
-    target_content = target_resource.content
-    relation_target_ids = target_resource.relations.get(relation_name)
+    target_content = target_resource.get("content")
+    if not target_content:
+        raise EvitableTransformationError()
```

Comment:
```suggestion
    try:
        target_content = target_resource["content"]
    except KeyError:
        raise EvitableTransformationError()
```
This should achieve the same, if you just want to wrap the `KeyError` and don't have the `KeyError` stacktrace attached

Reply:
The behavior is not the same. It does not handle the case where the value of `target_content` is an empty dictionary. However, `target_content` does not have to be a non-empty dictionary anyways. But if the data transformation relies on a certain property on it, the code does not catch it early enough. This opened a can of worms.

Reply:
Changed the behavior so that it accepts empty target_content